components/script/dom/htmlanchorelement.rs

```diff
@@ -667,7 +669,15 @@ pub fn follow_hyperlink(subject: &Element, hyperlink_suffix: Option<String>) {
         };
 
         // Step 14
-        debug!("following hyperlink to {}", url);
-        target_window.load_url(url, replace, false, referrer, referrer_policy);
+        let target = Trusted::new(target_window);
+        let task = task!(navigate_follow_hyperlink: move || {
+            debug!("following hyperlink to {}", url);
+            target.root().load_url(url, replace, false, referrer, referrer_policy, None);
+        });
+        target_window
+            .task_manager()
+            .dom_manipulation_task_source()
```

Comment:
should this be the networking task source?

Reply:
The DOM manipulation is the right one: https://html.spec.whatwg.org/multipage/links.html#following-hyperlinks:dom-manipulation-task-source